redfishMockupServer.py

```diff
@@ -430,6 +432,92 @@ def do_POST(self):
                                     sys.stdout.flush()
                                 self.send_response(204)
                                 self.event_id = self.event_id + 1
+                    elif 'TelemetryService/Actions/TelemetryService.SubmitTestMetricReport' in rpath:
+                        eventpath = os.path.join(apath, 'redfish/v1/EventService/Subscriptions', 'index.json')
+                        if self.server.shortForm:
+                            eventpath = eventpath.replace('redfish/v1/', '')
+                        success, jsonData = get_cached_link(eventpath)
+                        print(eventpath)
+                        if not success:
+                            # Eventing not supported
+                            self.send_response(404)
+                        else:
+                            # Check if all of the parameters are given
+                            if ('Name' not in data_received) or ('Value' not in data_received):
```

Comment:
Should be "MetricReportName" and "MetricReportValues". Looking further at the TelemetryService, there seems to be a discrepancy in data types too that we need to look at in the forum...

Reply:
With the current schema definition, we could just generate a single value in a metric report; this may be okay. So, the POST would look like:
```
{
    "MetricReportName": "TestReport",
    "MetricReportValues": "6"
}
```

and the generated report would look like:
```
{
    "@odata.context": "/redfish/v1/$metadata#MetricReport.MetricReport",
    "@odata.type": "#MetricReport.v1_0_0.MetricReport",
    "@odata.id": "/redfish/v1/TelemetryService/MetricReports/TestReport",
    "Id": "TestReport",
    "Name": "TestReport",
    "ReportSequence": "<SomeSequenceNumberIncremented>",
    "MetricReportDefinition": {
        "@odata.id": "/redfish/v1/TelemetryService/MetricReportDefinitions/TestReport"
    },
    "MetricValues": [
        "Timestamp": "<current time>",
        "MetricValue": "6"
    ]
}
```

@jcleung5549 was this the original intent, or is the action supposed to take in the entire array of values specified by "MetricValues"?